### ISSUE TITLE
Suggestion - Use "selector" only when needed in Custom CSS, not forced

### ISSUE THREAD

**Issue by pingram3541** [pojome/elementor#1086]
Currently when using the Pro Custom CSS feature we are restricted to css for that element and child selectors only.  We can't apply hierarchical rules such as "body.is-special .some-elementor-selector a" For example, adding custom css to a text element like so:

```
a {
  color: blue;
}
```

results in:
```
.unique-elementor-selector-class a {
  color: blue;
}
```

However we can already do this using "selector" to achieve the same result:
```
selector a {
  color: blue;
}
```

So why not remove the forced inclusion and allow us full freedom to write our own rules, media queries, @font rules and most importantly body class selectors so we can truly refine our designs without leaving the designer?

This way if we have full control without sacrificing the simplicity of using custom because we still have the dynamic "selector" variable. 

Yes we can do this with tools outside Elementor, but that's not what we want. =)

**Comment by pingram3541** [comment]
Another use case I just came across is when the design calls for absolute positioned elements I'd like to add rules such as:

```
body:not(.elementor-editor-active) .elementor-selector-section {
  position: absolute;
  ...
}
```

or:

```
body:not(.logged-in) .elementor-selector-section {
  position: absolute;
  ...
}
```

This way I get the desired results on the front end but do not break Elementor's inline editor controls mixed into the page content when editing.

**Comment by arielk** [comment]
Your right, we are going to improve this feature in the next version of E-pro..

**Comment by pingram3541** [comment]
Awesome, thanks for continuing to make this plugin amazing!

**Comment by arielk** [comment]
We released v1.0.5 please update and get it :)

**Comment by drawcard** [comment]
Legendary! Thank you for doing this work over the holiday break too...
